test/src/util/formatting/date-format.js

```diff
@@ -1,15 +1,15 @@
 import { expect } from 'chai';
-import { DateFormat } from 'util/formatting/date-format';
+import { DateFormat } from 'comp/util/date-format';
 
 describe('DateFormat', () => {
-    const dt = new Date(2020, 0, 2, 3, 4, 5, 6);
+    const dt = new Date.UTC(2020, 0, 2, 3, 4, 5, 6);
 
     it('should format date', () => {
-        expect(DateFormat.dStr(dt)).to.eql('2 Jan 2020');
+        expect(DateFormat.dStr(dt)).to.eql('Jan 2, 2020');
     });
 
     it('should format date and time', () => {
-        expect(DateFormat.dtStr(dt)).to.eql('2 Jan 2020 03:04:05');
+        expect(DateFormat.dtStr(dt)).to.eql('Jan 2, 2020, 03:04:05 AM');
```

Comment:
This test will fail if system locale is not `en-US`, it should set the locale now.

Reply:
But I see, en-us is the default, so it should work... sorry

Reply:
No, it doesn't fail. `en-US` is the default Locale provided by the application. I'm running an it-IT locale, still the test doesn't fail.